docs/src/development/sanitize-db.md

```diff
@@ -0,0 +1,135 @@
+---
+title: Sanitize databases
+description: Remove sensitive information from databases on non-production environments to control access.
+---
+
+When working on a new feature on your website, you want to use a new branch.
+Using a new branch makes sure that you don't risk breaking your live, production website.
+
+Creating a branch on Platform.sh copies both the code and the database to that new development branch.
+These code and database changes need to be tested before being merged into production.
+Depending on your processes, internal or external teams may interact with the development environment branch.
+
+Databases of live websites often contain personally identifiable information (PII)
+such as full names, mailing addresses, and phone numbers.
+To ensure people reviewing the code changes can't access information they shouldn't, sanitize the database of any PII that it may contain.
+
+## Before you begin
+
+You need:
+
+- A project with a [MySQL database](../configuration/services/mysql/_index.md).
+- if you aren't using Drupal, you will need the [Platform CLI](/development/cli/_index.md#cli-command-line-interface) installed locally.
+- if you are using Drupal, you will need [Drush](https://www.drush.org/latest/install/) installed.
+
+This guide is about sanitizing MySQL databases.
+
+This guide doesn't address:
+
+- Sanitizing NoSQL Databases ([MongoDB](../configuration/services/mongodb/_index.md), ...)
+- Input validation and input sanitization, which both help prevent security vulnerabilities
+
+## Sanitize the database
+
+Make sure that you only sanitize development environments and **never** the production environment.
+Otherwise you may lose most or even all of the relevant data stored in your database.
+
+First, take a [database dump](../configuration/services/mysql/_index.md#exporting-data) of your development environment.
+You won't alter your production data. This is just a safety precaution.
+To get a database dump, run: `platform db:dump -e <DEVELOPMENT_ENVIRONMENT_NAME>`.
+
+{{< codetabs >}}
+
+---
+title=Manually
+file=none
+highlight=false
+---
+
+Assumptions:
+
+- `user` is the table where all of your PII is stored.
+- `user` is a part of the `staging` development database.
```

Comment:
The SQL query is `SELECT * FROM users;`, which seems to indicate the table name is `users`.

It might also help to combine the two points into one since they relate to the same table. :shrug: